oshi-core/src/main/java/oshi/software/os/unix/freebsd/FreeBsdOSProcess.java

```diff
@@ -280,13 +280,18 @@ public long getContextSwitches() {
 
     @Override
     public boolean updateAttributes() {
-        String psCommand = "ps -awwxo state,pid,ppid,user,uid,group,gid,nlwp,pri,vsz,rss,etimes,systime,time,comm,majflt,minflt,args -p "
-                + getProcessID();
+        String psCommand =
```

Comment:
The original code did `+ getProcessID()` which compiles down to `StringBuilder` in the bytecode. The current version is roughly equivalent (two calls to `StringBuilder#append` rather than one). This isn't a regression, and I think the suggestion is an example of premature optimization. We're about to `fork` and `exec` which dwarfs any of these concerns.

Reply:
Fair point, but it does seem an easy fix with self-documenting code value.  We can address it in a later refactor though.